app/src/main/java/net/gsantner/opoc/util/Callback.java

```diff
@@ -56,4 +60,28 @@
     public interface b5<A, B, C, D, E> {
         boolean callback(A arg1, B arg2, C arg3, D arg4, E arg5);
     }
+
+    public interface s0 {
+        String callback();
+    }
+
+    public interface s1<A> {
+        String callback(A arg1);
+    }
+
+    public interface s2<A, B> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The interface name 's2' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5279643)